web/src/games/zooparade/components/bcard.tsx

```diff
@@ -26,11 +26,11 @@ interface InnerWrapper {
 
 export class BCard extends React.Component<InnerWrapper, {}> {
   render() {
-    var cardValue: string;
+    var cardDisplayValue: string;
     var image: any;
```

Comment:
Can you please change these "vars" to "let's"? Vars are globals and are a bad idea overall